recovery/translation_ko.ts

```diff
@@ -234,15 +235,15 @@ WARNING: This will erase all data on your SD card</source>
     <message>
         <location filename="initdrivethread.cpp" line="135"/>
         <source>SD card broken (writes do not persist)</source>
-        <translation>SD 카드가 손상되었습니다. (기록 내용은 유지되지 않음)</translation>
+        <translation>SD 카드가 손상되었습니다. (기록된 내용은 유지되지 않음)</translation>
     </message>
 </context>
 <context>
     <name>LanguageDialog</name>
     <message>
         <location filename="languagedialog.ui" line="135"/>
         <source>Language (l): </source>
-        <translation>언어 (l): </translation>
+        <translation>언어 (L): </translation>
```

Comment:
lowercase is wrong.
because Korean uses Hangul - is not Alphabet.
btw, Korean Keyboard Layout has Alphabet which is Uppercase.
https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRORT_kXh4k0YtfBIsabgNeEs8vUPZ8CSwVbMxWgHDQktr7UzpTKQ
lowercase l - be able to confusing for I when user see keyboard.

so, it must be change to uppercase.